x-pack/plugins/security_solution/common/api/entity_analytics/risk_engine/preview_route.schema.yaml

```diff
@@ -58,6 +58,11 @@ components:
           description: Defines the time period over which scores will be evaluated. If unspecified, a range of `[now, now-30d]` will be used.
         weights:
           $ref: '../common/common.schema.yaml#/components/schemas/RiskScoreWeights'
+        excludeAlertStatuses:
+          type: array
+          items:
+            type: string
```

Comment:
This refers to this field, yeah? https://github.com/elastic/kibana/blob/b169a9c729e71aa7d865dad5f87852c77941d5a4/x-pack/plugins/security_solution/common/api/model/alert.schema.yaml#L24-L30

We should probably use that component if so

Reply:
Sure, thanks!